src/Components/ServiceScene/Breakdowns/SearchInput.tsx

```diff
@@ -0,0 +1,25 @@
+import { css } from '@emotion/css';
+import { Icon, Input } from '@grafana/ui';
+import React, { HTMLProps } from 'react';
+
+interface Props extends HTMLProps<HTMLInputElement> {
+  onClear(): void;
+}
+
+export const SearchInput = ({ value, onChange, placeholder, onClear }: Props) => {
+  return (
+    <Input
+      value={value}
+      onChange={onChange}
+      suffix={value ? <Icon onClick={onClear} name={'x'} className={styles.clearIcon} /> : undefined}
```

Comment:
```suggestion
      suffix={value ? <Icon onClick={onClear} name={'times'} className={styles.clearIcon} /> : undefined}
```

times is usually used as clear icon.

Reply:
Yeah, it's way better. Thanks!